taskcluster/win-opt-base.tyml

```diff
@@ -0,0 +1,76 @@
+taskId: ${taskcluster.taskId}
+provisionerId: ${taskcluster.win.provisionerId}
+workerType: ${taskcluster.win.workerType}
+taskGroupId: ${taskcluster.taskGroupId}
+schedulerId: ${taskcluster.schedulerId}
+dependencies:
+  $map: { $eval: build.dependencies }
+  each(b):
+    $eval: as_slugid(b)
+created: { $fromNow: '0 sec' }
+deadline: { $fromNow: '1 day' }
+expires:
+  $if: 'event.event == "push"'
+  then: { $fromNow: '6 months' }
+  else: { $fromNow: '7 days' }
+
+extra:
+  github:
+    $if: 'event.event == "push"'
+    then: { $eval: taskcluster.github_events.merge }
+    else: { $eval: taskcluster.github_events.pull_request }
+
+routes:
+  $if: 'event.event == "push"'
+  then:
+    { $eval: build.routes }
+
+payload:
+  maxRunTime: { $eval: to_int(build.maxRunTime) }
+  mounts:
+    - file: msys2-base-x86_64.tar.xz
```

Comment:
Using mounts allows us to setup `msys2` within the build tree, and thus have task-specific package management. It's mostly used for tooling (tar / unzip / make / swig), so it should be pretty straight forward.
We rely on `.tar.xz` because the installer does no do unattended. Setting up with `choco` ties us to admin rights and it gets messy with pacman later (one can install but not uninstall some packages, etc., it ends up being system-wide so it may clash with different tasks)